Cms_frontend/src/data/dummy_structure.json

```diff
@@ -1,46 +1,55 @@
 {
     "root": [
         {
+            "id": 0,
```

Comment:
why is this file still necessary if theres a backend?

Reply:
Might as well keep around for test purposes? Will delete if it's not necessary though

Reply:
ahhh okay, I just thought your logic still implemented it,  but could probably move it in a test folder then so it isnt confusing

Reply:
It's best to remove this. We can interact directly with the DB for testing cases.